tests/PySys/tedge_connect_test_sm_services/run.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+
+"""
+Run connection test while being connected (positive case):
+
+Given a configured system with configured certificate
+When we setup EnvironmentC8y
+When we execute `sudo tedge connect c8y`
+When we validate stdout for Software Management Services started and enabled
+When we cleanup EnvironmentC8y
```

Comment:
I suppose those lines are executed in the `EnvironmentC8y` class but this is not clear from the test case at all, maybe a comment in the code or a note in the case description?

Reply:
added a comment.